WhiskyCmd/Main.swift

```diff
@@ -176,6 +176,7 @@ extension Whisky {
             let url = URL(fileURLWithPath: path)
             let program = Program(url: url, bottle: bottle)
             program.runInTerminal()
+            RunLoop.main.run()
```

Comment:
This never exists on my machine. I believe my original intention was to *not* have WhiskyCmd block for the entire lifetime of the `wineserver`, i.e. this should exit as soon as possible.

Reply:
This issue is consistently reproducible on my M1 Mac.

The execution timing of coroutine code is related to the scheduler (it doesn't start immediately like a thread). However, if the main thread exits too quickly, the coroutine code will not be executed. This is similar to writing the following code in Golang:

```golang
func main() {
    go func() {
        fmt.Println("hello")
    }()
}

There is a certain probability that "hello" will not be printed. 

You can try it on several computers.